WordPressCom-Stats-iOS/StatsEphemory.h

```diff
@@ -0,0 +1,15 @@
+#import <Foundation/Foundation.h>
+
+@interface StatsEphemory : NSObject
```

Comment:
Added some header documentation to accommodate the obscurity of the name. :smile: